eml/komand_eml/util/utils.py

```diff
@@ -1,88 +1,76 @@
 # Custom imports below
 import re
-import email
 from bs4 import UnicodeDammit
 
 
 def body(b, log):
     if b.is_multipart():
+        payload = None
         for part in b.walk():
-            ctype = part.get_content_type()
-            cdispo = str(part.get('Content-Disposition'))
+            content_type = part.get_content_type()
+            content_disposition = str(part.get('Content-Disposition'))
 
-            # skip any text/plain (txt) attachments
-            if ctype == 'text/plain' and 'attachment' not in cdispo:
+            if content_type == 'text/plain' and 'attachment' not in content_disposition:
                 try:
-                    body = part.get_payload(decode=True).decode('utf-8')  # decode
+                    payload = part.get_payload(decode=True).decode('utf-8')
                 except Exception as ex:
```

Comment:
can we raise a specific exception here? e.g keyerror typeerror ect
Same for all the generic Exceptions

Reply:
Here can be raised typeerror, UnicodeEncodeError. Some exceptions (i.e. LookupError) are catched and changed to some value. Method "get_payload" should raise `Exception` or its subclass.